oceannavigator/frontend/src/components/MapInputs.jsx

```diff
@@ -268,27 +218,17 @@ class MapInputs extends React.Component {
 
 //***********************************************************************
 MapInputs.propTypes = {
-  state: PropTypes.object,
```

Comment:
Remove a bunch of unused crap from the props and proptypes across several files.